packages/account/src/test-utils/launchNode.ts

```diff
@@ -265,21 +265,15 @@ export const launchNode = async ({
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const [realIp, realPort] = rowWithUrl.split(' ').at(-1)!.trim().split(':'); // e.g. "2024-02-13T12:31:44.445844Z  INFO new{name=fuel-core}: fuel_core::graphql_api::service: 216: Binding GraphQL provider to 127.0.0.1:35039"
 
-        // TODO: Remove delay after fuel-core issue is fixed
-        // https://github.com/FuelLabs/fuel-core/issues/2107
-        setTimeout(
-          () =>
-            // Resolve with the cleanup method.
-            resolve({
-              cleanup,
-              ip: realIp,
-              port: realPort,
-              url: `http://${realIp}:${realPort}/v1/graphql`,
-              snapshotDir: snapshotDirToUse as string,
-              pid: child.pid as number,
-            }),
-          500
-        );
+        // Resolve with the cleanup method.
+        resolve({
+          cleanup,
+          ip: realIp,
+          port: realPort,
+          url: `http://${realIp}:${realPort}/v1/graphql`,
+          snapshotDir: snapshotDirToUse as string,
+          pid: child.pid as number,
+        });
```

Comment:
I believe this can go after https://github.com/FuelLabs/fuels-ts/pull/3052, but it needs a changeset.

Reply:
Good catch @arboleya , turns out this may actually be useful for me in the [benchmarks PR](https://github.com/FuelLabs/fuels-ts/pull/3084/files#diff-b8f07cac51692505e88e065262fd373293a373ed5f6196eb13ec811e37aee136L281) as adding an intentional delay for certain tests.

Reply:
That's alright, but then it'll be intentional, whereas this one can still be reverted.

> ps.: Changeset still missing.